MVC_Activity01/MVC_Activity01/Controllers/HomeController.cs

```diff
@@ -9,17 +10,58 @@ namespace MVC_Activity01.Controllers
 {
     public class HomeController : Controller
     {
+       private CustomerContext db = new CustomerContext();
         public ActionResult Index()
         {
             ViewBag.Title = "Order Details";
-            var list = MockDb.Orders;
-            return View(list);
+            var customers = db.Customers.ToList();
+            var customerModel = customers.Select(c => new CustomerDTO
+            {
+                Id = c.Id,
+                Name = $"{c.Lastname}, {c.Firstname} {c.Middlename}"
+            }).ToList();
+
+            SelectList CustomerList = new SelectList(customerModel, "Id", "Name");
+            ViewData["CustomerList"] = CustomerList;
+            //  var list = MockDb.Orders;
+            return View();
```

Comment:
Orders should not be in the home page, create new page for this `/Orders`

Reply:
ok sir rey @rjs123431 noted sir.